data/topic-schema/src/main/resources/net/corda/schema/Flow.yaml

```diff
@@ -18,17 +25,24 @@ topics:
   FlowEventDLQTopic:
     name: flow.event.dlq
     consumers:
+      - flow
     producers:
+      - flow
     config:
   FlowMapperEventTopic:
     name: flow.mapper.event
     consumers:
+      - flow
     producers:
+      - flow
+      - rpc
```

Comment:
I guess p2p inbound is fetched via a process in flow processor so this is correct